### PR TITLE
Nightly

### PR DESCRIPTION
**Pull request by Mav7**

### CODE REVIEW

File: app/views/canvas/.#next.html.erb [hunk 1]
```diff
@@ -0,0 +1 @@
+thamav@thamav-Satellite-C855.3385:1446015693
```
Comment (skippyPeanutButter) [inline app/views/canvas/.#next.html.erb]: Is this a junk file? Why is this here?

**Comment by skippyPeanutButter** [comment]
The only issue I see is that when you click the next and prev  buttons it continues to decrement/increment the page count even if you are already at the first/last page of the comic. So if I click next two times when I'm at the last page I have to click prev 2 times if I want to go back to the previous page before the last. Is there another way we can implement this?

**Comment by Mav7** [comment]
I'm going to get to the bottom of this.

**Comment by skippyPeanutButter** [comment]
@Mav7 Does it make sense what I was saying? Have you tried this yourself?

**Comment by skippyPeanutButter** [comment]
Looks good to me, just some comments about the white space.

Nice work.
